afs-storage-api/src/main/java/com/powsybl/afs/storage/events/ParentChanged.java

```diff
@@ -9,34 +9,63 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import java.util.Objects;
+
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
 public class ParentChanged extends NodeEvent {
 
     public static final String TYPENAME = "PARENT_CHANGED";
 
+    @JsonProperty("oldParentId")
+    private final String oldParentId;
+
+    @JsonProperty("newParentId")
+    private final String newParentId;
+
     @JsonCreator
-    public ParentChanged(@JsonProperty("id") String id) {
+    public ParentChanged(@JsonProperty("id") String id, @JsonProperty("oldParentId") String oldParentId, @JsonProperty("newParentId") String newParentId) {
         super(id, TYPENAME);
+        Objects.requireNonNull(oldParentId);
+        Objects.requireNonNull(newParentId);
+        this.oldParentId = oldParentId;
+        this.newParentId = newParentId;
```

Comment:
This kind of lines should be written like this, to be more compact:
```
this.oldParentId = Objects.requireNonNull(oldParentId);
this.newParentId = Objects.requireNonNull(newParentId);
```

Reply:
indeed